src/python/roms/__init__.py

```diff
@@ -1,141 +1,95 @@
+from __future__ import annotations
+
+import base64
 import functools
-import logging
-import os
-import pathlib
-import sys
+import hashlib
+import json
+import tarfile
 import warnings
-from typing import List, Optional
-
-from ale_py import ALEInterface
-from ale_py.roms import plugins, utils
-
-# pylint: disable=g-import-not-at-top
-if sys.version_info >= (3, 9):
-    import importlib.resources as resources
-else:
-    import importlib_resources as resources
-# pylint: enable=g-import-not-at-top
-
-# Precedence is as follows:
-#  1. Internal ROMs
-#  2. External ROMs
-#  3. ROMs from atari-py.roms
-#  4. ROMs from atari-py-roms.roms
-_ROM_PLUGIN_REGISTRY: List[plugins.Plugin] = [
```

Comment:
Thanks for information, that is very helpful to know. We will look to see what we can do. 
If the ROMs are packaged inside the PyPI install, does that cause issues for google? 
The primarily issue I see is for users with additional external ROMs not included in AutoROMs standard set then I'm not sure how this would work with it.